tsdb/postings_for_matchers_cache.go

```diff
@@ -206,6 +267,13 @@ func (c *PostingsForMatchersCache) created(key string, ts time.Time, sizeBytes i
 		sizeBytes: sizeBytes,
 	})
 	c.cachedBytes += sizeBytes
+	span.AddEvent("recorded cached promise size", trace.WithAttributes(
```

Comment:
[nit] This seems like a more accurate description (to me at least)

```suggestion
	span.AddEvent("added cached value to expiry queue", trace.WithAttributes(
```